src/app/debug.tsx

```diff
@@ -135,7 +135,19 @@ const Debug = ( c: RenderContext ) => {
 				</figure>
 
 				<figure className="queue">
-					<p>Build Queue</p>
+					<p>Building</p>
+					{ pendingHashes.size ? (
+						<ul>
+							{ Array.from( pendingHashes ).map( hash => (
+								<li>{ hash }</li>
```

Comment:
```suggestion
								<li key={ hash }>{ hash }</li>
```

Reply:
added it to both sets of list items in https://github.com/Automattic/dserve/pull/98/commits/738aeb8bcc2d4e466c19a3edfbdedba8f82df866